packages/docregistry/src/context.ts

```diff
@@ -91,7 +91,6 @@ export class Context<
       path: this._path,
       contents: manager.contents
     });
-    this.model.sharedModel.setState('path', this._path);
```

Comment:
Where is the state initialized? In the backend?

Reply:
Yes, in the backend. And the state is only used in collaborative mode.